yb-voyager/src/compareperf/templates/performance_comparison_report.template

```diff
@@ -362,6 +381,9 @@
                         {{range .AllComparisons}}
                         <tr>
                             <td class="query-text" data-value="{{.Query}}" title="{{.Query}}">{{.Query}}</td>
+                            <!-- Performance Metrics -->
+                            <td data-value="{{if eq .MatchStatus "MATCHED"}}{{.ImpactScore}}{{else}}-1{{end}}">{{if eq .MatchStatus "MATCHED"}}{{printf "%.2f" .ImpactScore}}{{else}}N/A{{end}}</td>
+                            <td data-value="{{if eq .MatchStatus "MATCHED"}}{{.SlowdownRatio}}{{else}}-1{{end}}" style="border-right: 3px solid #1e8449;">{{if eq .MatchStatus "MATCHED"}}{{printf "%.2fx" .SlowdownRatio}}{{else}}N/A{{end}}</td>
```

Comment:
add line separation between these two columns

Reply:
adding it for other missing places also